src/auth.rs

```diff
@@ -19,6 +19,8 @@ use {
     url::Url,
 };
 
+pub const STATEMENT: &str = "I further authorize this app to send and receive messages on my behalf using my WalletConnect identity. Read more at https://walletconnect.com/identity";
```

Comment:
Did we decide on the faq or identity path eventually?

Reply:
`/identity` is the final path. I setup a redirect in https://github.com/WalletConnect/www-walletconnect/pull/104 which will redirect to the actual docs page after we update it.